authoring-new-actions.md

```diff
@@ -0,0 +1,130 @@
+# Process to publish GitHub Actions for Azure
+
+# Introduction
+
+[GitHub Actions](https://help.github.com/en/articles/about-github-actions) gives developers the flexibility to build an automated software development lifecycle workflow directly from their repositories. Since releasing Actions in November 2017, GitHub partners have contributed enormously to Marketplace, so teams can extend and automate workflows with their existing tools.
+
+Microsoft [announced](https://azure.microsoft.com/en-us/blog/github-actions-for-azure-is-now-generally-available/) the general availability of [GitHub Actions for Azure](https://github.com/Azure/actions) for creating workflows to package, release, and deploy apps to the cloud from GitHub. [GHActions\_vTeam](mailto:GHActions_vTeam@microsoft.com) has been formed to spearhead this effort of collaborating with various product teams at Microsoft to release these actions.
+
+Also, at [Microsoft Build 2020](https://aka.ms/MS-build-session-Continuous-Delivery) we announced that GitHub Actions for Azure are now integrated into Visual Studio Code, Azure CLI, and the Azure Portal simplifying the experience of [deploying to Azure from your favorite tools.](https://azure.microsoft.com/en-us/blog/deploy-to-azure-using-github-actions-from-your-favorite-tools/)
+
+This document is intended to help Microsoft teams and developers with the recommended process to create and publish more actions to GH marketplace. Please reach out to the [vTeam](mailto:GHActions_vTeam@microsoft.com) for any queries.
+
```

Comment:
This statement is odd, it introduces words like teams, marketplace, create and publish. Lets be clear what the goal of this document is. Also, is this a binding recommendation?

Reply:
Clarified the taxonomy by including the links in the introduction section.